### DISCUSSION TITLE
RocketFuel Theme 5.2.1 released

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#4557] [Show and tell]
RocketFuel Theme 5.2.1 released. Download now from the Oqtane Marketplace
![image](https://github.com/user-attachments/assets/f1686b2c-bea0-4f1e-9811-97d872b34a42)

**Comment by SSFlyer01** [comment]
This theme only works in Interactive/WASM?

**Reply by leigh-pointer** [reply]
@SSFlyer01 The theme is an interactive theme "Studio-Elf Rocket Fuel [Interactive]" and will only work when the site settings are set to Interactive.

![image](https://github.com/user-attachments/assets/d70b17b9-c71b-4f9a-b991-a1fc867f5f5b)

**Comment by SSFlyer01** [comment]
I understand the interactive, but I can't get it to work with SignalR, only WASM.

**Reply by leigh-pointer** [reply]
studio-elf.net is running RockerFuel Interactive with SignalR.  There should be nothing else to configure.  Do you have SignalR running on other blazor sites?
